addons/towing/functions/fnc_isSuitableSimulation.sqf

```diff
@@ -19,8 +19,7 @@ params ["_target"];
 
 // need toLower since apparently this isn't case sensitive
 private _simulationType = getText ((configOf _target) >> "simulation");
-TRACE_1("sim type",_simulationType);
+// TRACE_1("sim type",_simulationType);
 
 // Biki lies, you can both tow and tow as either TankX or CarX
-(toLower _simulationType) in ["tankx", "carx"]
-
+(toLower _simulationType) in ["tankx", "carx", "shipx", "helicopterrtd"]
```

Comment:
Ship and Helicopter have no support for setTowParent, the wiki is correct on that.

But ships don't have wheel brakes and steering (which is what setTowParent does)
And RTD helicopter also does some special funky stuff which is why it works, though does it also work with advaned helisim thing turned off?

Reply:
I only tested with disabled AFM. Can't test with enabled one for several weeks. So I hope someone will test it while reviewing.